packages/bingo/src/cli/getTemplatePackageData.test.ts

```diff
@@ -18,7 +18,52 @@ vi.mock("read-package-up", () => ({
 	},
 }));
 
+let isWindowsPaths = false;
+
+vi.mock("node:url", async () => {
+	const nodeUrl = await vi.importActual<typeof import("node:url")>("node:url");
+	return {
+		...nodeUrl,
+		fileURLToPath: (url: string | URL) =>
+			nodeUrl.fileURLToPath(url, { windows: isWindowsPaths }),
+	};
+});
+
+vi.mock("node:path", async () => {
+	const nodePath =
+		await vi.importActual<typeof import("node:path")>("node:path");
+	const nodePathWindows =
+		await vi.importActual<typeof import("node:path/win32")>("node:path/win32");
+	const nodePathPosix =
+		await vi.importActual<typeof import("node:path/posix")>("node:path/posix");
+	return {
+		default: {
+			...nodePath,
+			dirname: (path: string) =>
+				isWindowsPaths
+					? nodePathWindows.dirname(path)
+					: nodePathPosix.dirname(path),
+		},
+	};
```

Comment:
[Testing] Refactoring nitpick, we can shrink this to be a bit less syntax-heavy:

```suggestion
	const nodePathWindows =
		await vi.importActual<typeof import("node:path/win32")>("node:path/win32");
	const nodePathPosix =
		await vi.importActual<typeof import("node:path/posix")>("node:path/posix");
	return {
		default: {
			...(await vi.importActual("node:path")),
			dirname: (path: string) =>
				isWindowsPaths
					? nodePathWindows.dirname(path)
					: nodePathPosix.dirname(path),
		},
	};
```